examples/helloworld/source/app.d

```diff
@@ -31,16 +31,35 @@ void main (string[] args)
     {
         defaultGetoptPrinter("Some information about the program.",
             helpInformation.options);
+        return 0; // Not an error, so exit normally
     }
 
-    // An `Exception` will be thrown if this fails
-    auto config = clargs.parseConfigFile!Config;
+    try
+    {
+        // An `Exception` will be thrown if this fails
+        auto config = clargs.parseConfigFile!Config;
 
-    for (size_t i = 0; config.limit == 0 || i < config.limit; ++i)
+        for (size_t i = 0; config.limit == 0 || i < config.limit; ++i)
+        {
+            writeln("Hello World to you ", config.name);
```

Comment:
Doesn't it print the same `config.name` in the for loop?

Reply:
It does, why ?

Reply:
Ah, I thought the `for` loop prints the same thing. there's something I need to know about `Config`. Let me check the class more.

Reply:
This code is just an example, so it doesn't do anything really logical. The main point is to show how a user would parse a config file.